picard/mbxml.py

```diff
@@ -320,6 +350,33 @@ def recording_to_metadata(node, m, track=None):
         elif name == 'video' and nodes[0].text == 'true':
             m['~video'] = '1'
     m['~length'] = format_time(m.length)
+    # Analyse to see if we should populate the movement variables
+    if m.partsof:
+        target_work = None
+        for work, parts_of in m.partsof.items():
+            works = set([p['workid'] for p in parts_of])
+            if target_work is None:
```

Comment:
I did this deliberately as empty set means no common target i.e. the Union operations in the loop eventually result in an empty set - so needed something else to differentiate.